### DISCUSSION TITLE
Showing file name instead of program name on taskbar

### DISCUSSION THREAD

**Discussion by rik1301** [sqlitebrowser/sqlitebrowser#3100] [Q&A]
Hi folks.

I have 3 separate .sqbpro databases which I often run side by side to work on (all unrelated to each other, so no I can't just move all the tables into 1 database).  Is there a way to change the taskbar name from "DB Browser to SQLite" to the actual file names?  If so, how please?  Having all 3 instances with the same name is rather annoying as I have to remember which one is which.

Thanks. :)

**Comment by chrisjlocke** [comment]
Doesn't DB4S show the filename/project filename in the caption?

![image](https://user-images.githubusercontent.com/7872218/180229569-6d64a5d1-b1b8-40b6-8763-fc9101e983b6.png)

**Reply by chrisjlocke** [reply]
![image](https://user-images.githubusercontent.com/7872218/180229732-c73d1911-6fa2-4f62-97f8-57838579df62.png)

**Reply by rik1301** [reply]
Yes, that's exactly what I DON'T want to show.  I want the file name to be visible on the taskbar tab rather than the long "DB Browser for SQLite" which obviously doesn't tell me which database I'm looking at unless I hover my mouse over it for 2-3 seconds each time.  I want it to instead show the database names, eg. test1db, test2db in place of "DB Browser for SQLite".  Any way to do that please?

**Reply by chrisjlocke** [reply]
I'm guessing there is a Windows tool to change the Windows caption - its just a property after all.  I don't think DB4S regularly changes it, nor internally uses it (ie, gets the Window caption to determine what database you have open).

![image](https://user-images.githubusercontent.com/7872218/180265261-a59ad41e-c874-421e-a8b2-0127257ac69a.png)

As an aside, notepad (in my taskbar in the above example) is the 5th icon.  There are 4 Notepads in that group.  If I press [Win]+5 (Windows key and number 5 on the keyboard) the first notepad window is displayed ([Win]+4 would show the 4th application in the taskbar, ie, Task Manager).  Press [Win]+5 again, and the second instance of Notepad pops up.  Point is, if you know the database you're working on is the 3rd instance, and the other one is the first instance, its easy to quickly press [Win]+5 the necessary amount of times... 
Its not a solution to your problem (I'll hunt around for a caption tweaker) but could be a handy workaround...

**Reply by rik1301** [reply]
Thanks Chris.  Yes, if you or anyone else knows of a way to change the "caption" that would be fantastic.  For clarity, I don't have my taskbar set up like you do with icons.  I have mine set to not group the same program types, and have them all next to each other, pretty much for the reasons stated as I can quickly navigate between them and see what they all are at a glance.  Eg. currently I have :

eM Client
this Chrome instance "Showing file name..."
an incognito Chrome instance open for something else
"DB Browser for SQLite" for db 1
"DB Browser for SQLite" for db 2
"DB Browser for SQLite" for db 3
